azure-kusto-ingest/azure/kusto/ingest/_ingest_client.py

```diff
@@ -1,5 +1,3 @@
-"""Kusto ingest client for Python."""
```

Comment:
Doesn't it cause pylint to tell you that you need to write a module dicstring?

Reply:
It does actually, but since it wasn't very helpful I removed it. I can make sure all files have them . not sure they have any value (as classes and functions are documented).

Other azure projects tend to just put license information instead...